src/generators/model/default/model.php

```diff
@@ -236,11 +236,7 @@ public static function <?php echo $column_data['func_opts_name']?>()
         return [
 <?php
         foreach($column_data['values'] as $k => $value){
-            if ($generator->enableI18N) {
```

Comment:
@eluhr This is BC, right? the `enableI18N` check is done in `generateString()` in the `Generator`.